webrtc.html

```diff
@@ -3932,52 +3901,178 @@ <h4 id="sec.identity-proxy-communications">Identity Provider
+            use.</p>
+
+            <p>The IdP selects the identity to assert.  The optional
+            <var>usernameHint</var> parameter is the same value that was pass to
+            <code>setIdentityProvider</code>.</p>
+
+            <p>The IdP provides a promise that resolves to an
+            <code><a>RTCIdentityAssertionResult</a></code> to successfully
+            generate an identity assertion. Any other value, or a rejected
+            promise, is treated as an error.</p>
+          </dd>
+
+          <dt>Promise&lt;RTCIdentityValidationResult&gt;
+          validateAssertion(DOMString assertion, DOMString origin)</dt>
+          <dd>
+            <p>A user agent invokes this method on the IdP to request the
+            validation of an identity assertion.</p>
+
+            <p>The <var>assertion</var> parameter includes the assertion that was
+            recovered from an <code>a=identity</code> in the session description;
+            that is, the value that was part of the
+            <code><a>RTCIdentityAssertionResult</a></code> provided by the IdP
+            that generated the assertion.</p>
+
+            <p>The <var>origin</var> parameter identifies the origin of the
+            <code><a>RTCPeerConnection</a></code> that triggered this request.  An
+            IdP can use this information as input to policy decisions about
+            use.</p>
+
+            <p>The IdP returns a Promise that resolves to an
+            <code><a>RTCIdentityValidationResult</a></code> to successfully
+            validate an identity assertion and to provide the actual identity.
+            Any other value, or a rejected promise, is treated as an error.</p>
+          </dd>
+        </dl>
+      </section>
+
+      <section>
+        <h2>Identity Assertion and Validation Results</h2>
+
+        <dl class="idl" data-merge="RTCIdentityProviderRegistrar"
+            title="dictionary RTCIdentityAssertionResult">
+          <dt>required RTCIdentityProviderDetails idp</dt>
+          <dd>
+            <p>An IdP provides these details to identify the IdP that validates
+            the identity assertion.  This struct contains the same information
+            that is provided to <code>setIdentityProvider</code>.</p>
+          </dd>
+
+          <dt>required DOMString assertion</dt>
+          <dd>
+            <p>An identity assertion.  This is an opaque string that SHOULD
+            contain all information necessary to assert identity.  This value is
+            consumed by the validating IdP.</p>
+          </dd>
+        </dl>
+
+        <dl class="idl" data-merge="RTCIdentityProviderRegistrar"
+            title="dictionary RTCIdentityProviderDetails">
+          <dt>required DOMString domain</dt>
+          <dd>
+            <p>The domain name of the IdP that validates the associated identity
+            assertion.</p>
+          </dd>
+
+          <dt>DOMString protocol = "default"</dt>
```

Comment:
The default corresponds to the default on `setIdentityProvider`.  It allows the default identity provider to have a slightly more compact encoding for their assertions.